tests/s2_inference/test_generic_clip_model.py

```diff
@@ -369,25 +369,6 @@ def test_vectorise_without_clip_type(self):
         vectorise(model_name, content, model_properties, device="cpu")
 
 
-    def test_validate_model_properties_unknown_model_error(self):
-        """_validate_model_properties should throw an error if model is not in registry,
-            and if model_properties have not been given in index
-        """
```

Comment:
isn't this test still valid (even if it needs some updating)?

Reply:
it is an exact duplicate of `test_validate_model_properties_unknown_model_error` in `test_generic_model.py`